python/eos/plot/plotter.py

```diff
@@ -108,14 +110,17 @@ def setup_plot(self):
             self.ax.xaxis.set_ticks_position('both')
 
             if 'scale' in myx:
+                myxscale = myx['scale']
+
+            if 'scaling_factor' in myx:
 
                 if 'format' in myx:
                     self.xformat = myx['format']
                 else:
                     self.xformat = '${x}$'
                     eos.warn("Argument plot:x:format might be required when using plot:x:scale to avoid side effects")
 
-                self.xscale = float(myx['scale'])
+                self.xscale = float(myx['scaling_factor'])
```

Comment:
The code would be clearer if this variable was named `x_scaling_factor`.

Reply:
Yes definitely